cumulusci/tasks/bulkdata/load.py

```diff
@@ -139,7 +140,10 @@ def _run_task(self):
                                 f"Step {after_name} did not complete successfully: {','.join(result.job_errors)}"
                             )
         if self.options["set_recently_viewed"]:
-            self._set_viewed()
+            try:
+                self._set_viewed()
+            except Exception as e:
+                self.logger.warning(f"Could not set recently viewed because {e}")
```

Comment:
Glad you added this, since this is the first time the org schema cache will get more widespread use.

Reply:
4f5ff9bd3